examples/ctap2.rs

```diff
@@ -129,6 +129,10 @@ fn main() {
                     panic!("Unexpected error: {:?}", e)
                 }
             },
+            Ok(StatusUpdate::PinAuthInvalid) => {
```

Comment:
Shouldn't this be an error, not OK?

Reply:
No. In this case `Ok()` means 'we managed to receive something' and `Err()` means "the channel for transmitting StatusUpdates is broken'. It does not reflect on the meaning of the status update.